djangoproject/templates/includes/footer.html

```diff
@@ -29,6 +29,7 @@ <h2>Get Involved</h2>
             <li><a
               href="{% url 'document-detail' lang='en' version='dev' url="internals/security" host 'docs' %}#reporting-security-issues">Report
               a Security Issue</a></li>
+            <li><a href="/foundation/individual-members/">Individual membership</a></li>
```

Comment:
I'd use the URL name here instead of hardcoding the link:
```suggestion
            <li><a href="{% url 'individual-members' %}">Individual membership</a></li>
```